tests/test_vision.py

```diff
@@ -3,12 +3,8 @@
 from fastai.vision import *
 
 @pytest.fixture(scope="module")
-def path(request):
+def path():
     path = untar_data(URLs.MNIST_TINY)
-    d = defaults.device
```

Comment:
referring: https://github.com/fastai/fastai/pull/1000#discussion_r228928591
@sgugger : I had removed the device dependency existing in `test_vision.py` tests (it was present from before)

but right now on checking more, it seems like @jph00 had pushed that change few weeks back
commit: https://github.com/fastai/fastai/commit/d343a36caadc97214f7276f69d2413888a448b5e

Marking him, may be there was some specific thought behind it - we can revert the current commit if that was the case

Reply:
Okay just discussed with Jeremy in person - he was checking some specific CPU compatibility issue at that time - we don't need it anymore
Cc @sgugger 
(keeping the commit, PR is ready for review)

Reply:
Ignore my above comment - we do that need those **particular tests** that way - Jeremy just clarified 

I m going to rename this file to *_cpu* to avoid confusion